plexapi/media.py

```diff
@@ -124,7 +124,41 @@ def audioStreams(self):
     def subtitleStreams(self):
         """ Returns a list of :class:`~plexapi.media.SubtitleStream` objects in this MediaPart. """
         return [stream for stream in self.streams if stream.streamType == SubtitleStream.STREAMTYPE]
-
+        
+    def setDefaultAudioStream(self, stream):
+        """ Set the default :class:`~plexapi.media.AudioStream` for this MediaPart.
+
+            Parameters:
+                stream (:class:`~plexapi.media.AudioStream`): AudioStream to set as default 
+            
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: If stream is not an AudioStream.
+        """
+        if type(stream) == AudioStream:
```

Comment:
I think it's better to use `isinstance()` here, instead of direct types comparison, just in case if somebody suddenly create own implementation of `AudioStream`